source/code/scxsystemlib/common/scxostypeinfo.cpp

```diff
@@ -52,7 +52,7 @@ using namespace SCXCoreLib;
 
 namespace {
 
-#if defined(linux)
+#if defined(linux) && !defined(PF_DISTRO_ULINUX)
```

Comment:
If you're sure it's never called, then I'm okay with this change. But we still need to deal with what you're doing with the InstallBuilder ...